AUTHORS

```diff
@@ -42,6 +42,7 @@ By order of apparition, thanks:
     * Shaung Cheng (S3 docs)
     * Andrew Perry (Bug fixes in SFTPStorage)
     * Manuel Kaufmann (humitos)
+    * Andrii Skoryk (Google Cloud Storage)
 
```

Comment:
Are you sure I don't have to add Pozotron instead? If not, then I can also add you, as you've done a lot here too :+1:

Reply:
Nah, this has been your project :)